.circleci/organise_doc.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const fs = require('fs')
+const args = require('minimist')(process.argv.slice(2))
+
+// get the output directory
+const typedocJSON = require('../typedoc.json')
+const typedocOutputDir = typedocJSON.out
+
+// root dir is the absolute path to use, provided by CI
+const rootDir = args.rootdir
+if (rootDir === '' || rootDir == null){
```

Comment:
this type of lines can be replaces by `!var` as empty string, null and undefined evaluate to false. For example: `(rootDir === '' || rootDir == null)` becomes `!rootDir`

Reply:
Cool it did not know that! Thanks.